internal/operators/ocs/validations.go

```diff
@@ -0,0 +1,133 @@
+package ocs
+
+import (
+	"encoding/json"
+	"math"
+
+	"github.com/docker/go-units"
+	"github.com/openshift/assisted-service/internal/host"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+//go:generate mockgen -source=validations.go -package=ocs -destination=mock_validations.go
+type OcsValidator interface {
+	ValidateOCSRequirements(cluster *models.Cluster) bool
```

Comment:
once we will change the code to be more generic - we would like that each operator will return not only the bool value but also a reason to the user that will help him to solve the validation error - I'll update you when it will be ready.